### PR TITLE
Updating readme

### PR DESCRIPTION
**Pull request by prafullchvn**
Updated the documentation for project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 This is readme of TestingDeleteToday
 Added this line the second time
-Again some changes
\ No newline at end of file
+Adding this line for getting reviewed
```
Comment (prafullchvn) [inline README.md:5]: I think we should remove this line

**Review by ritikabhosale — CHANGES_REQUESTED** [review]
Consider removing the last line

**Review by prafullchvn — COMMENTED** [review]
this is review by prafull

File: README.md (hunk 1 above)
Comment (prafullchvn) [reply README.md:5]: don't resolve it just yet

File: README.md [hunk 2]
```diff
@@ -2,4 +2,6 @@
...
+One more line
+Again some changes
```
Comment (prafullchvn) [inline README.md:7]: I think we need one more of this line

**Review by prafullchvn — COMMENTED** [review]
Suggestion in line changes.

**Review by ritikabhosale — APPROVED** [review]
All good

File: README.md [hunk 3]
```diff
@@ -2,4 +2,6 @@
 
 This is readme of TestingDeleteToday
```
Comment (prafullchvn) [inline README.md:3]: this is very good line

File: README.md [hunk 4]
```diff
@@ -2,4 +2,6 @@
...
 Added this line the second time
```
Comment (prafullchvn) [inline README.md:4]: this is also very good line

**Review by prafullchvn — COMMENTED** [review]
What a lines. Maja aa gaya

File: README.md (hunk 4 above)
Comment (prafullchvn) [inline README.md:4]: this is another line that I like

**Review by prafullchvn — COMMENTED** [review]
this is a line I liked

**Pull request closed without merging**